README.md

```diff
@@ -44,19 +44,37 @@ To build the operator:
     make build
     ```
 
-## Updating Custom Resource Definitions
-
-The custom resource definitions used by the Verrazzano Operator are auto generated by the Verrazzano project `verrazzano-crd-generator`.
-The steps to include changes made to the custom resource definitions are:
-
-* Make the API changes in the `verrazzano-crd-generator` project
-* Tag a branch of `verrazzano-crd-generator`, for example `0.n`
-* Edit `go.mod` to use the newly tagged API changes.  For example `github.com/verrazzano/verrazzano-crd-generator v1.2.0`
+## Updating Dependencies
+
+The dependencies used by the Verrazzano Operator are managed by go's `go.mod` file.
+Any license changes related to the new dependency versions should be carefully evaluated as discussed in the "Introducing a new dependency" section.  
+The steps to include changes made to dependencies are:
+
+* Run `go get` to update references to the dependency in the `go.mod` file.
+  There are three options for doing this:
+  1. Update to a specific tagged version of the dependency.  For example:
```

Comment:
1. Update to a specific tagged version... ->  (use sub-bulleted items for this list) Update to a specific tagged version...

Reply:
Fixed.